src/js/edu-benefits/components/EducationView.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { formatReviewDate } from '../../common/schemaform/helpers';
+
+export default function EducationView({ formData }) {
+  let from = '';
+  let to = '';
+  if (formData.dateRange) {
+    from = formatReviewDate(formData.dateRange.from);
+    to = formatReviewDate(formData.dateRange.to);
+  }
+
+  return (
+    <div>
```

Comment:
OK, but that is very vague :)
E.g., which of these fields should be included in the summary:
school name, city, state, dates attended, hours completed & hour type, degree/cert earned, major course of study

Right now I've added name and dates, but that was an arbitrary choice.